source/renderer/app/containers/wallet/WalletSettingsPage.js

```diff
@@ -29,6 +42,7 @@ export default class WalletSettingsPage extends Component<Props> {
       profile,
     } = this.props.stores;
     const activeWallet = wallets.active;
+    const activeWalletPublicKey = wallets.activePublicKey;
```

Comment:
Can you change it to 

`const { activePublicKey } = wallets;` and than forward it like:

`walletPublicKey={activePublicKey}`

Reply:
@DeeJayElly It is following line 44 naming to make them consistent.